src/plugins/proxy/src/plugin.cpp

```diff
@@ -283,9 +283,13 @@ ov::Any ov::proxy::Plugin::get_property(const std::string& name, const ov::AnyMa
         // ov::device::id changes the default proxy device
         if (std::find(supported_prop.begin(), supported_prop.end(), ov::device::id) == supported_prop.end())
             supported_prop.emplace_back(ov::device::id);
-        if (std::find(supported_prop.begin(), supported_prop.end(), CONFIG_KEY_INTERNAL(CONFIG_DEVICE_ID)) ==
+        return supported_prop;
+    } else if (name == ov::internal::supported_properties) {
```

Comment:
Above we also have: `if (name == CONFIG_KEY_INTERNAL(CONFIG_DEVICE_ID))`
Could you please remove legacy property?

Reply:
> Above we also have: `if (name == CONFIG_KEY_INTERNAL(CONFIG_DEVICE_ID))` Could you please remove legacy property?

All occurrences of `CONFIG KEY_INTERNAL(CONFIG _DEVICE_ID)` were replaced